docs/conversation.md

```diff
@@ -4,6 +4,63 @@
 * Base endpoint for API v2 is: `/ocs/v2.php/apps/spreed/api/v2`
 * Base endpoint for API v3 is: `/ocs/v2.php/apps/spreed/api/v3`
 
+## Get listed conversations
+
+* Method: `GET`
+* Endpoint: `/listed-room`
+
+* Response:
+    - Status code:
+        + `200 OK`
+        + `401 Unauthorized` when the user is not logged in
+
+    - Header:
+
+        field | type | Description
+        ------|------|------------
+        `searchTerm` | string | search term
+
+    - Data:
```

Comment:
Use this instead of duplicating it:
```suggestion
    - Data: See array definition in `Get user´s conversations`
```

We do already for some other things too and the difference should be 0